### ISSUE TITLE
Testing code gives an error for the model loaded

### ISSUE THREAD

**Issue by Avi-141** [IDKiro/action-recognition-pytorch#6]
Expected 4-dimensional input for 4-dimensional weight [64, 3, 11, 11], but got 3-dimensional input of size [3,224, 224] instead
```
original_model=models.alexnet(pretrained=True)
lstm_layers=2
num_classes=2
hidden_layers=512
fc=4096
model = LSTMModel(original_model,"alexnet",num_classes,lstm_layers,hidden_layers,fc)
optimizer = torch.optim.Adam([{'params': model.fc_pre.parameters()},
                                {'params': model.rnn.parameters()},
                                {'params': model.fc.parameters()}],
                                1e-4)
checkpoint=torch.load(PATH)
model.load_state_dict(checkpoint['state_dict'])
optimizer.load_state_dict(checkpoint['optimizer'])
epoch=checkpoint['epoch']
model.eval()


test_transformer = transforms.Compose([
            transforms.Resize((h,w)),
            transforms.ToTensor(),
            transforms.Normalize(mean, std),
            ]) 
```
 
```
def predict_image(image):
    image_tensor = test_transformer(image).float()
    image_tensor = image_tensor.unsqueeze_(0)
    input_var = Variable(image_tensor)
    input_var = input_var.to(device)
    print(input_var.shape)
    output = model(input_var)
    #print(output)
    index = output.data.cpu().numpy().argmax()
    return index
```

Please let me know the steps further,

**Comment by IDKiro** [comment]
tensor should be 4-dim: N (batch), C (channel), H (height), W (width).
function `unsqueeze()` will help you.
you'd better search this kind of questions on stackoverflow / google.

**Comment by Avi-141** [comment]
I have trained my datasets using your instructions and have also ```unsqueezed``` as you can see in the ```predict_image``` function. If you could provide some other insight it would be really helpful, since i have already googled and searched but haven't found any solutions that helps me.

**Comment by IDKiro** [comment]
```
print(input_var.shape)
```
what's the output

**Comment by Avi-141** [comment]
hello, it is a list. Hence the length comes out to be 49.
